ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -861,6 +861,39 @@
               recipients: projectantrea-dev@googlegroups.com rahulj@vmware.com
           triggers: []
           wrappers: []
+      - 'cloud-{name}-{test_name}':
```

Comment:
Can you check that there is any setting the job template level to prevent this job from running in parallel? If there is no such setting, we want to limit the parallelism to 1.

Because if there are two or more instances of this job running at the same time, it may deploy too many VMs.

Reply:
Yes, we can add `concurrent: false` to limit parallelism for this job,